app/services/articles/feeds/basic.rb

```diff
@@ -19,6 +19,9 @@ def default_home_feed(**_kwargs)
         return articles unless @user
 
         articles = articles.where.not(user_id: UserBlock.cached_blocked_ids_for_blocker(@user.id))
+        if (anti_tags = @user.cached_antifollowed_tag_names).any?
+          articles = articles.not_cached_tagged_with_any(anti_tags)
+        end
```

Comment:
This is a good catch! It's confusing, and I confused myself. Should be fixed now in 844697f5e. 🤞 

I would note — as at least part of what's confusing myself here, and I am maybe not alone — that there's something of a plague of poor naming: `large_forem_experimental` (the admin setting) does _not_ (in the case of a signed-in user invoke `LargeForemExperimental` (the class). _That_ class is only invoked by [`assign_feed_stories`](https://github.com/forem/forem/blob/main/app/controllers/stories_controller.rb#L250-L253), which is only invoked for [logged-out users](https://github.com/forem/forem/blob/main/app/controllers/stories_controller.rb#L136).

In _this_ case (because antitags necessarily involve a signed-in user, we are in [`signed_in_base_feed`](https://github.com/forem/forem/blob/main/app/controllers/stories/feeds_controller.rb#L38-L48), which is then either a `Articles::Feeds::Basic` or `Articles::Feeds.feed_for` — in which case, we wind up in `Articles::Feeds::VariantQuery`. (Which does _not_, in any way that I can see, actually inherit from `Basic`.)

Also potentially confusing for me & maybe other folks: the toggle between `large_forem_experimental` and `basic` is under "feed **strategy**", not "feed **style**".

Reply:
I have just fallen down a `VariantQuery` rabbit hole, but this does work 🎉